tests/conftest.py

```diff
@@ -1,135 +1,190 @@
 """Pytest test config tools."""
-from typing import Dict, Any, Optional, List
-import json
+import os
+import tarfile
+from typing import List, Optional
 import pytest
 from pathlib import Path
 
-from disease.database import Database
-from disease.schemas import Disease
+from disease.database import AWS_ENV_VAR_NAME, Database
+from disease.etl.base import Base
+from disease.query import QueryHandler
+from disease.schemas import Disease, MatchType, MatchesKeyed
+
+
+def pytest_collection_modifyitems(items):
+    """Modify test items in place to ensure test modules run in a given order.
+    When creating new test modules, be sure to add them here.
```

Comment:
I had thought in a different project we made a check to ensure len(MODULE_ORDER) == len(# of files that start with test_)

Reply:
Good catch! I think we did that in anyvar.